x-pack/plugins/observability/public/pages/slos/components/slo_summary.tsx

```diff
@@ -22,18 +20,12 @@ export interface Props {
 }
 
 export function SloSummary({ slo, historicalSummary = [], historicalSummaryLoading }: Props) {
-  const { uiSettings } = useKibana().services;
-  const percentFormat = uiSettings.get('format:percent:defaultPattern');
+  const { currentValue, sloTarget, errorBudgetRemaining } = useSLOSummary(slo);
```

Comment:
I would maybe rename it to something that conveys the formatting behaviour, e.g. 
```suggestion
  const { sliValue, target, errorBudgetRemaining } = useSloFormattedSummary(slo);
```